### PR TITLE
REMOVE MGT

### PR DESCRIPTION
**Pull request by ColonelPanicks**
Addresses #30.

### CODE REVIEW

File: lib/cloudware/domain.rb [hunk 1]
```diff
@@ -55,15 +54,15 @@ def load_cloud
 
     def describe
       @describe ||= begin
-                    domain = Struct.new :name, :region, :provider, :networkcidr, :prvsubnetcidr, :mgtsubnetcidr
-                    @describe = domain.new(get_item('domain'), get_item('region'), get_item('provider'), get_item('network_cidr'), get_item('prv_subnet_cidr'), get_item('mgt_subnet_cidr'))
+                    domain = Struct.new :name, :region, :provider, :networkcidr, :prvsubnetcidr
+                    @describe = domain.new(get_item('domain'), get_item('region'), get_item('provider'), get_item('network_cidr'), get_item('prv_subnet_cidr')
```
Comment (mjtko) [inline lib/cloudware/domain.rb]: Missing trailing-`)` :)

**Comment by WilliamMcCumstie** [comment]
I don't believe this is the approach we should take for removing the management network. The extend of the changes in your single commit could silently break any number of commands. Instead we should work through each command taking a systematic approach. That way it will be easier to diagnose any issues in the future.

Remember that we do not have `travis` nor a complete test suite behind us. Thus we are going to have to rely on `git bisect` to diagnose issues.

**Comment by mjtko** [comment]
I think, in this case, it's clear from reading the source what the changes are here.

There needs to be a balance between pragmatism and idealism while we move this tool from prototype to release, and waiting for Travis and/or test suites before we make sweeping changes is going to significantly slow things down when we need them to be fast.

I think the pragmatic thing to do here is for this PR to be reopened, the single-character mistake corrected and merged through.  Tests can come later, more important to get the features knocked off right now. :)

**Comment by WilliamMcCumstie** [comment]
Okay, hopefully it is working. I have rebased on #42 and resolved the conflicts. Travis should also start behaving as well. Ignore `CodeClimate`, it is just progressively finding code duplication.

**Comment by WilliamMcCumstie** [comment]
The following items still have `mgt` networks in them. Some maybe a result of my rebasing (not sure), however most of the references concern the templates.
```
[root@controller cloudware]# git grep -l mgt                                   
README.md
lib/cloudware/cli.rb
lib/cloudware/commands/machine/create.rb
providers/aws/templates/domain.yml
providers/aws/templates/machine-master.yml
providers/aws/templates/machine-slave.yml
providers/azure/templates/domain.json
providers/azure/templates/machine-master.json
providers/azure/templates/machine-slave.json
```

**Comment by WilliamMcCumstie** [comment]
Can we also look into what is causing the error bellow. I think it might be related to the above templates. I have tracked the issue back to: 6ee9a2df582508db56f25281b4313f3c31b132db
```
[root@controller cloudware]# bin/cloud domain create pickled-pumpkin2 -p aws -r
 eu-west-1 --trace                                                             
[OK]  Verifying mgt subnet CIDR is valid         
[OK]  Checking domain name is valid                                  
[OK]  Verifying provider is valid                       
[OK]  Creating new deployment            
/tmp/cloudware/lib/cloudware/aws.rb:244:in `rescue in deploy': Parameters: [mgtSubnetCidr] must have values (RuntimeError)
        from /tmp/cloudware/lib/cloudware/aws.rb:237:in `deploy'
        from /tmp/cloudware/lib/cloudware/aws.rb:213:in `create_domain'
        from /tmp/cloudware/lib/cloudware/domain.rb:64:in `create'
        from /tmp/cloudware/lib/cloudware/commands/domain/create.rb:33:in `block in run'
        from /tmp/cloudware/lib/cloudware/command.rb:48:in `block in run_whirly'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/gems/whirly-0.2.6/lib/whirly.rb:171:in `start'
        from /tmp/cloudware/lib/cloudware/command.rb:46:in `run_whirly'
        from /tmp/cloudware/lib/cloudware/commands/domain/create.rb:32:in `run'
        from /tmp/cloudware/lib/cloudware/command.rb:13:in `run!'
        from /tmp/cloudware/lib/cloudware/cli.rb:51:in `block in action'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/command.rb:219:in `call'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/patches/validate_inputs.rb:29:in `call'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/command.rb:187:in `run'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/patches/validate_inputs.rb:15:in `run'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/runner.rb:477:in `run_active_command'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/runner.rb:66:in `run!'
        from /opt/cloudware/opt/ruby/lib/ruby/gems/2.4.0/bundler/gems/commander-4090e181cf28/lib/commander/delegates.rb:16:in `run!'
        from bin/cloud:27:in `<main>'
```

**Comment by ColonelPanicks** [comment]
I have addressed the issue and opened a new PR #51, this PR still needs to be merged or more rebasing will ruin our lives.

**Pull request merged**